camera_name: azure_kinect
image_width: 1280
image_height: 720

camera_matrix:
  fx: 990.33284
  fy: 990.73059
  cx: 1023.22204
  cy: 771.07537